### PR TITLE
Add NelmioApiDocBundle 3.0 support

### PR DESCRIPTION
**Pull request by GuilhemN**
| Q             | A
| ------------- | ---
| License       | MIT

Fixes https://github.com/symfony/recipes-contrib/issues/197 and https://github.com/symfony/recipes-contrib/pull/207

### CODE REVIEW

File: nelmio/api-doc-bundle/3.0/config/routes/nelmio_api_doc.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+NelmioApiDocBundle:
+    # Requires the Asset component and the Twig bundle
+    resource: "@NelmioApiDocBundle/Resources/config/routing/swaggerui.xml"
+    prefix:   /api/doc
```
Comment (dbu) [inline nelmio/api-doc-bundle/3.0/config/routes/nelmio_api_doc.yaml]: should this include a commented out definition for exposing the swaggerdoc json as well? (it should not be on by default, but having it in here seems useful to me)

    path: /api/doc.json
    methods: GET
    defaults: { _controller: nelmio_api_doc.controller.swagger }

Comment (GuilhemN) [reply nelmio/api-doc-bundle/3.0/config/routes/nelmio_api_doc.yaml]: Sure, added :)

File: nelmio/api-doc-bundle/3.0/config/routes/nelmio_api_doc.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Requires the Asset component and the Twig bundle
+app.swagger_ui:
+    path: /api/doc
+    methods: GET
+    defaults: { _controller: nelmio_api_doc.controller.swagger_ui }
+
+# app.swagger:
```
Comment (dbu) [inline nelmio/api-doc-bundle/3.0/config/routes/nelmio_api_doc.yaml]: i think this should have a comment above this line what it does

Comment (GuilhemN) [reply nelmio/api-doc-bundle/3.0/config/routes/nelmio_api_doc.yaml]: added :)

**Pull request merged**